coffee-chats/src/main/java/com/google/step/coffee/servlets/ChatRequestServlet.java

```diff
@@ -36,8 +42,10 @@ private ChatRequest buildChatRequestFromHttpRequest(HttpServletRequest request)
     int minPeople = Integer.parseInt(getParameterOrDefault("minPeople", request, "1"));
```

Comment:
Haven't noticed it before, but passing a non-integer `minPeople` will throw an unhandled `NumberFormatException`.

Reply:
I have added some extra handling for other types too. Done.